static/app/components/replays/replayPlayer.tsx

```diff
@@ -63,13 +57,13 @@ function BasePlayerRoot({
   // Update the scale of the view whenever dimensions have changed.
   useEffect(() => {
     if (viewEl.current) {
-      const scale = flexibleHeight
-        ? Math.min((windowDimensions?.width || 0) / videoDimensions.width, 1)
-        : Math.min(
-            (windowDimensions?.width || 0) / videoDimensions.width,
-            (windowDimensions?.height || 0) / videoDimensions.height,
+      const scale = fixedHeight
+        ? Math.min(
+            windowDimensions.width / videoDimensions.width,
+            windowDimensions.height / videoDimensions.height,
             1
-          );
+          )
+        : Math.min(windowDimensions.width / videoDimensions.width, 1);
```

Comment:
inverted the variable name, from `const flexibleHeight = !fullscreen` now we have `const fixedHeight = fullscreen`. So the condition is flipped too